src/main/java/MethodOptionsTwoHandler.java

```diff
@@ -0,0 +1,15 @@
+public class MethodOptionsTwoHandler implements Handler {
+
+    private static final String METHOD_OPTIONS_TWO_URL = "/method_options2";
+    private static final String OPTIONS_REQUEST = "OPTIONS";
+
+    public String handle(RequestParser request) {
```

Comment:
There seems to be a common pattern in how these requests are handled. If that proves to be more common across other handlers then can maybe consider looking into the template pattern